Changes

```diff
@@ -1,3 +1,167 @@
+- VDB (#2373) :
+  - Added native support for representing VDB volume data in Gaffer
+    scenes.
+  - The standard SceneReader node can now load .vdb files.
+  - VDB objects can be rendered directly to Arnold.
+  - The Viewer now provides basic visualisation of VDB grids.
+  - The SceneInspector provides inspection of grid metadata etc
+    in the Object section.
+  - Several new nodes demonstrate the feasibility of manipulating
+    VDB grids directly in Gaffer's node graph :
+      - LevelSetToMesh
+      - MeshToLevelSet
+      - LevelSetOffset
+- Catalogue : Added a new CatalogueSelect node. This can be placed
+  downstream of a Catalogue to override the image to view (#2370).
+- UI Editor : Added the ability to add custom buttons to a plug (#2348).
+- Gadget : Introduced layer-based rendering for Gadget-based UI
+  elements(#2304).
+
+Improvements
+------------
+
+- OpenColorIO nodes : Added role presets to all colorspace plugs (#2379)
+- NodeGraph : Added icons to signify that a node has been bookmarked (#2369)
+- PlugLayout : Improved layout of successive accessory widgets (#2348). They
+  are now all placed in the same row.
+- Simplified 3delight configuration based on #DELIGHT environment
+  variable (#2311).
+- TaskPlug/FilterPlug/ShaderPlug : Tightened `acceptsInput()` constraints (#2321).
+- OSLObject : Added support for reading and writing matrix primitive variables (#2327).
+- GafferImage : Moved default format plug creation to gui config (#2333).
+
+Fixes
+-----
+
+- Catalogue :
+  - Fixed bug that prevented images with a '.' in the filename from
+    being loaded (#2370).
+  - Fixed crashes caused by removing several images in quick
+    succession (#2337).
+- File Menu : Fixed "Import..." bugs (#1077, #2339).
+- Edit Menu : Fixed automatic layout of pasted nodes (#2383)
+- ShaderPlug : Fixed bug when deserialising inputs from BoxIO nodes (#2374)
+- Crop : Fixed bug whereby changing the format plug value did not trigger
+  recomputation (#2368).
+- ImageWriter (#2364) :
+  - Fixed a crash when writing a data window with a single pixel .
+  - Fixed writing of empty data windows to write a single pixel data
+    window instead of a full image. OpenEXR does not support empty data
+    windows, so we must write at least one pixel.
+- NodeGraph :
+  - Fixed confusing Box navigation behaviour (#2343, #2347).
+  - Fixed overlay text to always be on top (#674, #2304).
+  - Fixed layering so dragged connections are always drawn behind
+    nodes, and highlighted plugs are drawn above all others (#2304).
+- SubGraph : Accounted for BoxIO nodes in `correspondingInput() (#2360).
+  This fixe a node reconnection bug when deleting a Box or Reference, and
+  allows Box and Reference nodes to be drag-inserted onto existing connections.
+- ScriptNode :
+  - Fixed node deletion problems caused by errors in `correspondingInput()`
+    implementations (#2355). This was preventing nodes representing missing Arnold
```

Comment:
issue number at the end?

Reply:
There are a bunch more of these btw. I didn't comment them all because I thought it would be annoying to get so many comments.